### PR TITLE
Move Resource Printing Page from SIG CLI guides to the kubectl reference

### LINKED ISSUE #45660: Move Resource Printing from SIG CLI guides to the kubectl reference
**This is a Feature Request**

**What would you like to be added**

Move the pages under `Resource Printing` from the [SIG CLI guides](https://kubectl.docs.kubernetes.io/guides) to the [upstream kubectl reference](https://kubernetes.io/docs/reference/kubectl)

**Why is this needed**

Part of https://github.com/kubernetes/website/issues/25943. The source code to move resides [here](https://github.com/kubernetes-sigs/cli-experimental/tree/master/site/content). You can refer to https://github.com/kubernetes/website/issues/45600 to see how it can be done.

**Comments**

/sig docs
/language-en
/help-wanted

### PR DESCRIPTION
**Pull request by prashantrewar**
Fixes #45660

### CODE REVIEW

File: content/en/docs/reference/kubectl/resource_printing/cluster_information.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+
+{{< alert color="success" title="TL;DR" >}}
+- Print information about the Cluster and Client versions
+- Print information about the Control Plane
+- Print information about Nodes
+- Print information about APIs
+{{< /alert >}}
+
+# Cluster Info
```
Comment (sftim) [inline content/en/docs/reference/kubectl/resource_printing/cluster_information.md:18]: Thanks.
We've realized that a straight conversion isn't right. This proposed tweak should help a lot though:
```suggestion
```

File: content/en/docs/reference/kubectl/resource_printing/describe.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+description: >
+    Describes current resources
+---
+
+
+{{< alert color="success" title="TL;DR" >}}
+- Print verbose debug information about a Resource
+{{< /alert >}}
+
+# Describe Resources
```
Comment (sftim) [inline content/en/docs/reference/kubectl/resource_printing/describe.md:15]: Thanks.
We've realized that a straight conversion isn't right. This proposed tweak should help a lot though:
```suggestion
```

File: content/en/docs/reference/kubectl/resource_printing/fields.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+---
+
+
+
+{{< alert color="success" title="TL;DR" >}}
+- Format and print specific fields from Resources
+- Use when scripting with Get
+{{< /alert >}}
+
+# Print Resource Fields
```
Comment (sftim) [inline content/en/docs/reference/kubectl/resource_printing/fields.md:17]: Thanks.
We've realized that a straight conversion isn't right. This proposed tweak should help a lot though:
```suggestion
```

File: content/en/docs/reference/kubectl/resource_printing/queries_and_options.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+---
+
+
+{{< alert color="success" title="TL;DR" >}}
+- Queries for Getting or Describing Resources
+{{< /alert >}}
+
+# Matching Objects from Get and Describing
+
+## Motivation
```
Comment (sftim) [inline content/en/docs/reference/kubectl/resource_printing/queries_and_options.md:17]: Thanks.
We've realized that a straight conversion isn't right. This proposed tweak should help a lot though:
```suggestion
```

File: content/en/docs/reference/kubectl/resource_printing/raw.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+{{< alert color="success" title="TL;DR" >}}
+- Get or List Raw Resources in a cluster as Yaml or Json
+{{< /alert >}}
+
+# Print Raw Resource
+
+## Motivation
+
+Inspecting or Debugging Resources.
```
Comment (sftim) [inline content/en/docs/reference/kubectl/resource_printing/raw.md:19]: Thanks.
We've realized that a straight conversion isn't right. This proposed tweak should help a lot though:
```suggestion
```

File: content/en/docs/reference/kubectl/resource_printing/summary.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+{{< alert color="success" title="TL;DR" >}}
+- Get a Summary of Resources Running in the Cluster
+{{< /alert >}}
+
+# Summarizing Resources
+
+## Motivation
+
+Quickly summarizing a collection of Resources and their state.
```
Comment (sftim) [inline content/en/docs/reference/kubectl/resource_printing/summary.md:18]: Thanks.
We've realized that a straight conversion isn't right. This proposed tweak should help a lot though:
```suggestion
```

**Review by sftim — COMMENTED** [review]
This move is less straighforward than we thought it would be. Sorry about that.

Try these tweaks.

File: content/en/docs/reference/kubectl/resource_printing/cluster_information.md (hunk 1 above)
Comment (prashantrewar) [reply content/en/docs/reference/kubectl/resource_printing/cluster_information.md]: Done!

File: content/en/docs/reference/kubectl/resource_printing/fields.md (hunk 3 above)
Comment (prashantrewar) [reply content/en/docs/reference/kubectl/resource_printing/fields.md]: Done!

File: content/en/docs/reference/kubectl/resource_printing/queries_and_options.md (hunk 4 above)
Comment (prashantrewar) [reply content/en/docs/reference/kubectl/resource_printing/queries_and_options.md]: Done!

**Comment by prashantrewar** [comment]
> This move is less straighforward than we thought it would be. Sorry about that.
> 
> Try these tweaks.

I made the changes that you suggested.

@sftim, Thank you for the help and suggestion.

**Pull request closed without merging**